sway/sway-output.5.scd

```diff
@@ -190,6 +190,19 @@ must be separated by one space. For example:
 	may have no effect or produce unexpected output when used together with future
 	HDR support features.
 
+*output* <name> allow_tearing yes|no
+	Allows or disallows screen tearing as a result of asynchronous page flips,
+	and an immediate presentation mode from a client.
+
+	With asynchronous page flips, frames from the client are presented as soon
+	as possible instead of synchronizing with the monitor's vblank interval 
+	(VSync). This prevents stutter and reduces latency in games.
```

Comment:
"This prevents stutter and reduces latency in games."

This is pretty subjective and debatable, so I'd prefer to leave out that part.

Reply:
Removed.